### PR TITLE
[Ready for Review] Unapologetic Grid Check Buffs

### PR DESCRIPTION
**Pull request by LordFowl**
When deprived of power airlocks will now open, unless they are marked as secure (Vault airlocks, AI core airlocks, and EVA airlocks) or are welded/bolted.
When deprived of power blast doors will automatically close.
Blast doors can no longer be opened with just a crowbar.
Some mapping changes have been made to ensure no department is completely inescapable due to blast doors.

No longer will people who had the misfortune of not shoving a crowbar in their pocket be forced to stare at an empty wall for 40 minutes. On the flipside now power outages will not just be a free ticket to loot the armory/bridge.

### CODE REVIEW

**Comment by flimango** [comment]
Sounds cool. The only issue I foresee is literally the entire station venting if an area loses pressure when a grid check hits, if someone has broken an external airlock and disabled the air alarm, or such.

**Comment by flimango** [comment]
The grid check would disable the air alarms too though, wouldn't it? So only one room would need to be breached without firelocks.

**Comment by Alberyk** [comment]
Elevators doors should also not open if there is no power, due to obvious reasons.

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -1311,6 +1318,13 @@ About the new airlock wires panel:
 		// If we lost power, disable electrification
 		// Keeping door lights on, runs on internal battery or something.
 		electrified_until = 0
+		//if we lost power open 'er up
+		if(insecure)
+			open(1)
```
Comment (HeyBanditoz) [inline code/game/machinery/doors/airlock.dm]: Could also `unlock(1)` so it unbolts (by force.)

Comment (LordFowl) [reply code/game/machinery/doors/airlock.dm]: That is not desired behavior.

**Comment by skull132** [comment]
Z7 conflicted. Plz clear.

**Pull request merged**